docs/reference/meson-compatibility.rst

```diff
@@ -0,0 +1,50 @@
+.. SPDX-FileCopyrightText: 2023 The meson-python developers
+..
+.. SPDX-License-Identifier: MIT
+
+.. _meson-compatibility:
+
+*******************
+Meson compatibility
+*******************
+
+``meson-python`` tightly integrates with Meson to produce Python
+wheels ans sdists. Therefore, correct operation depends on
```

Comment:
typo in "ans"

Reply:
I really need to setup my editor to spell check `.rst` files...